### ISSUE TITLE
Can not set up Dwarf Canon near the Dwarf Engineer

### ISSUE THREAD

**Issue by Kenix3** [Marwolf/Open-RSC#40]
- special message setting up dwarf canon near engineer ... 78x95 tiles
it is not permitted to set up a cannon...
...this close to the dwarf black guard

**Comment by LionRsc** [comment]
According to wiki that is correct behavior. [I have no idea what tiles are included/excluded and what tiles you're referring to]
"The cannon cannot be setup near[Clarify] Nulodion, if a player attempts to do so the following message is displayed:"
http://prntscr.com/kgzzge
http://runescapeclassic.wikia.com/wiki/Dwarf_cannon

**Comment by twothousandandone** [comment]
This is correct behavior and has been documented via replays, and discussed on the RSC Preserve discord.

Replays should be referenced here
